ms2query/ms2library.py

```diff
@@ -0,0 +1,332 @@
+from typing import List, Dict, Union
+import pandas as pd
+import numpy as np
+from tqdm import tqdm
+from tensorflow.keras.models import load_model as load_nn_model
+from gensim.models import Word2Vec
+from matchms.similarity import CosineGreedy, ModifiedCosine
+from matchms.Spectrum import Spectrum
+from ms2deepscore.models import load_model as load_ms2ds_model
+from ms2deepscore import MS2DeepScore
+from spec2vec import Spec2Vec
+from spec2vec.vector_operations import cosine_similarity_matrix
+from ms2query.query_from_sqlite_database import get_spectra_from_sqlite
+from ms2query.app_helpers import load_pickled_file
+from ms2query.spectrum_processing import create_spectrum_documents
+
+
+class Ms2Library:
```

Comment:
We anyway sometimes violate some naming conventions :) 
But to me it should be `MS2Library` because `MS2` refers to MS/MS = fragmentation mass spectrometry.